### ISSUE TITLE
Skin laceration

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10701]
**Preferred term label:**
Skin laceration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Cut or defect through skin, surgical or traumatic

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Extensor tendon laceration, hand/wrist
Flexor tendon injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001125
- label: Skin laceration
- parents: [Localized skin lesion (HP:0011355)](https://hpo.jax.org/app/browse/term/HP:0011355)
- pmid: PMID:31424750
- def: An injury in which skin and underlying tissues are cut or torn.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10701
- synonyms:
